### DISCUSSION TITLE
Translate .srt (and maybe other subs) from the UI

### DISCUSSION THREAD

**Discussion by rowanfuchs** [lingarr-translate/lingarr#20] [Ideas & Feature requests]
[DarkShadowSwE](https://github.com/DarkShadowSwE) commented [2 days ago](https://github.com/lingarr-translate/lingarr/issues/16#issue-2568727750)
Hi,

If possible.

I would like to just upload a srt file and have it translated.

Just a from and to language dropdown selector and a file upload form.

**Comment by exatobg** [comment]
Hello, this would be a good step in the direction of future integration between lingarr and [morpheus65535/bazarr](https://github.com/morpheus65535/bazarr) . So that bazarr can use lingarr as subtitle source. For when subtitles aren't available in a required language.

**Comment by rowanfuchs** [comment]
I had envisioned that Bazarr downloads the source subtitles and from that Lingarr is able to translate to a target that Bazarr can't find. But for some people all the service integration is more than they actually need so a seperate translatable upload to download could suffice.

**Comment by a-camacho** [comment]
I would also love to just upload a srt file and translate it.
Does not matter how.

Thanks a lot, just upvoted.

**Comment by rowanfuchs** [comment]
Several users of Lingarr also use [SubtitleEdit](https://github.com/SubtitleEdit/subtitleedit/) it looks like this provides a auto translate with even more services but from a desktop ui
